protobuf/proto/agent.proto

```diff
@@ -75,38 +75,159 @@ message ListSecretsRequest {
  * Deploys a single container
  *
  */
-message DeployRequest {
+message Environment { repeated string env = 1; }
+
+message InstanceConfig {
+  /*
+  containerPreName, mapped into host folder structure,
+  used as namespace id
+  */
+  string prefix = 1;
+  optional string mountPath = 2; // mount path of instance (docker only)
+  optional Environment environment = 3; // environment variable map (piped)
+  optional string repositoryPrefix = 4; // registry repo prefix
+}
+
+message RegistryAuth {
+  string name = 1;
+  string url = 2;
+  string user = 3;
+  string password = 4;
+}
+
+message ContainerStateItem {
+  string containerId = 100;
+  string name = 101;
+  string command = 103;
+  google.protobuf.Timestamp createdAt = 104;
+  /* The 'State' of the container (Created, Running, etc) */
+  common.ContainerState state = 105;
+  /*
+   * The 'Status' of the container ("Created 1min ago", "Exited with code 123",
+   * etc). Unused but left here for reverse compatibility with the legacy
+   * version.
+   */
+  string status = 106;
+  string imageName = 107;
+  string imageTag = 108;
+
+  repeated Port ports = 1000;
+}
+
+message ContainerStateListMessage {
+  optional string prefix = 100;
+  repeated ContainerStateItem data = 1000;
+}
+
+// Container config
+
+message Port {
+  int32 internal = 100;
+  int32 external = 101;
+}
+
+message PortRange {
+  int32 from = 100;
+  int32 to = 101;
+}
+
+message PortRangeBinding {
+  PortRange internal = 100;
+  PortRange external = 101;
+}
+
+message Volume {
+  string name = 100;
+  string path = 101;
+  optional string size = 102;
+  optional common.VolumeType type = 103;
+  optional string class = 104;
+}
 
+message VolumeLink {
+  string name = 100;
+  string path = 101;
+}
+
+message InitContainer {
+  string name = 100;
+  string image = 101;
+  optional bool useParentConfig = 102;
+
+  repeated VolumeLink volumes = 1000;
+  repeated string command = 1001;
+  repeated string args = 1002;
+  map<string, string> environments = 1003;
+}
+
+message ImportContainer {
+  string volume = 100;
+  string command = 101;
+
```

Comment:
Inconsistent newline between simple and repeated values. (It's minor but we have to fix this in the following PRs)

Reply:
Same with the one-line and multiple-line comments.